### PR TITLE
Update Profiler UI for Symfony 6.2+.

### PR DESCRIPTION
**Pull request by nickygerritsen**
This is based on the closed PR #1558, but only does two changes:

* It updates the icon in 6.2+.
* It adds the metric groups.

I will do a separate PR to add the tabs.

Screenshot on 6.1:
image

And on 6.2:
image

### CODE REVIEW

**Comment by nickygerritsen** [comment]
Not sure why the test fails btw. If I add `$twigLoaderFilesystem->addPath(__DIR__ . '/../Resources/views', 'Doctrine');` to the `setUp` method it works, but we already load other resources in the twig...?

File: Resources/views/Collector/db.html.twig
Comment (ostrolucky) [inline Resources/views/Collector/db.html.twig]: Would be nice to know why are we replacing UTF-8 character with SVG, which is larger and harder to change.

File: Resources/views/Collector/db.html.twig
Comment (ostrolucky) [inline Resources/views/Collector/db.html.twig]: Not sure why are we changing position of this section, this makes the PR quite hard to review

File: Resources/views/Collector/db.html.twig
Comment (ostrolucky) [inline Resources/views/Collector/db.html.twig]: why are these headings removed?

File: Resources/views/Collector/db.html.twig
Comment (ostrolucky) [inline Resources/views/Collector/db.html.twig:121]: Why was this moved under? Seems quite random change to me.

File: Resources/views/Collector/db.html.twig
Comment (ostrolucky) [inline Resources/views/Collector/db.html.twig]: This also seem quite arbitrary decision to me to hide the section when count is 0

File: Resources/views/Collector/db.html.twig
Comment (ostrolucky) [inline Resources/views/Collector/db.html.twig]: these can be simplified like so
```twig
<span class="icon">{{ include('@Doctrine/Collector/' ~ (profiler_markup_version < 3 ? 'icon' : 'database') ~ '.svg') }}</span>
```

**Comment by nickygerritsen** [comment]
I changed the PR quite a bit and only updated the icon and added the metric groups. Once this is merged we can work from there to add the tabs for the different things.

I also didn't drop any metrics or changed any order anymore.

File: Resources/views/Collector/db.html.twig
Comment (nickygerritsen) [reply Resources/views/Collector/db.html.twig]: Agreed, got rid of this.

File: Resources/views/Collector/db.html.twig
Comment (nickygerritsen) [reply Resources/views/Collector/db.html.twig:121]: Agreed, reverted.

File: Resources/views/Collector/db.html.twig
Comment (nickygerritsen) [reply Resources/views/Collector/db.html.twig]: I think because the UTF-8 characters don't look nice. But I also didn't really like the arrows, so I leave this to someone with actual UX/UI skills.

File: Resources/views/Collector/db.html.twig
Comment (ostrolucky) [inline Resources/views/Collector/db.html.twig:106]: What are these for? Is it so that it stretches to full width?

Comment (nickygerritsen) [reply Resources/views/Collector/db.html.twig:106]: It “groups” the metrics. Look at the screenshots: in 6.1 it is 4 blocks, in 6.2 it is 4 parts of a bigger block. Without it, there would be whitespace between the blocks

**Review by ostrolucky — APPROVED** [review]

**Pull request merged**
